readme.md

```diff
@@ -86,6 +86,7 @@ If you want to contribute, please read the [contribution guidelines](contributin
 
 ### Code Quality
```

Comment:
I'm not sure that it's the best section for this plugin, but I don't see good alternatives.